src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/SkeletonNode.cs

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Generic;
+
+namespace Microsoft.SqlTools.ServiceLayer.ExecutionPlan.Contracts
+{
+    /// <summary>
+    /// A simpler version of a node because of the reduced amount of properties.
+    /// </summary>
+    public class SkeletonNode
```

Comment:
Can we rename this to ExecutionGraphComparisonResult or ExecutionGraphMatchingResult? 
The name SkeletonNode does not seem intuitive.